### ISSUE TITLE
Unable to install on Py3.8, 3.9 - version not found for matching distribution

### ISSUE THREAD

**Issue by tshrjn** [s0d3s/PyAudioWPatch#14]
[comment]: <> (Just paste "x" inside brackets, for example: - [x] Some positive statement)

**What problem are you facing?**
Unable to install the package via pip: `pip install PyAudioWPatch`

Error thrown is as:
```
ERROR: Could not find a version that satisfies the requirement PyAudioWPatch (from versions: none)
ERROR: No matching distribution found for PyAudioWPatch
```

The issue can be recreated as follows:
```
conda create -n test python=3.9
conda activate test
pip install PyAudioWPatch
```

**Comment by shosseini811** [comment]
Same here

**Comment by shosseini811** [comment]
Its for Windows ONLY.

"
Wheels are available for Windows, Python 3.{7,8,9,10,11}.
All wheels support APIs: WMME, WASAPI, DirectX(DSound)."

**Comment by s0d3s** [comment]
Yes, unfortunately, the library is currently `only available` under Windows.
Support for other systems **may appear** no earlier than in 5 months.

**Comment by DoramisStudio** [comment]
I am using Windows but i get the same error too, any solution?
![image](https://github.com/s0d3s/PyAudioWPatch/assets/69141236/f04a2807-d196-40d7-9b3f-040d1177815e)

**Comment by DoramisStudio** [comment]
@s0d3s any solution? currently I am using Windows 11 Pro

**Comment by s0d3s** [comment]
@DoramisStudio What version of Python do you use?(`python -V`)
